torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -463,6 +463,7 @@ struct Environment {
                "__round__",
                std::make_shared<BuiltinFunction>(aten::round, at::nullopt))},
           {"hash", std::make_shared<BuiltinFunction>(aten::hash, at::nullopt)},
+          {"id", std::make_shared<BuiltinFunction>(prim::id, at::nullopt)},
```

Comment:
Since this doesn't desugar to anything other than an op, this should go in `_builtins.py` instead of inside the compiler

Reply:
It’s generally a good property to have things parse strings as well

Reply:
How is this case different from `list` in https://github.com/pytorch/pytorch/pull/34112#discussion_r393371705?

Reply:
Sorry, i wasn't saying that `list` should be registered as a compiler builtin, just that it could be. I don't know what the exact line we want to draw here is, but I know there are plans to register custom gradients use the C++ parser, so python builtin keywords would probably be nice to support (such as `list`).